twitchio/ext/commands/bot.py

```diff
@@ -329,10 +336,6 @@ async def handle_commands(self, message):
             The message object to get content of and context for.
 
         """
-        original_content = message.content
-
-        if "reply-parent-msg-id" in message.tags:
-            message.content = message.content.split(" ", 1)[1]
         context = await self.get_context(message)
         await self.invoke(context)
 
```

Comment:
original_content is not defined so this line can be removed

Reply:
Oh, there went something wrong while merging the changes from this repo.